app/src/main/java/com/android/sample/resources/dummydata/E2E_DummyData.kt

```diff
@@ -112,12 +112,18 @@ private val e2e_activity2 =
         participants = listOf(e2e_user1),
         comments = listOf(e2e_comment1))
 
-val defaultUserCredentials =
+val defaultUserCredentials1 =
     mapOf(
         "email" to e2e_user1_email,
         "password" to e2e_user1_password,
         "first name" to e2e_user1.name,
         "full name" to "${e2e_user1.name} ${e2e_user1.surname}")
+val defaultUserCredentials2 =
```

Comment:
In that case I wouldn't name it defaultUser, but nvm it is not really important.